scripts/polska-scena/core/SquirrelSystemBundle/Resources/Methods.nut

```diff
@@ -0,0 +1,67 @@
+const STRING = "string";
+const INT = "integer";
+const FLOAT = "float";
+const ARRAY = "array";
+
+/**
+ * @var string separator
+ * @var array array
+ * @return string
+ */
+function implode(separator, array)
+{
+    local text;
+    foreach(elemnt in array){
+        text += separator + element
+    }
+    return text;
+}
+
+
+/**
+ * @var mixed needle
+ * @var array haystack
+ * @var bool strict
+ */
+function in_array(needle,haystack, strict = false)
+{
+    try {
+        if (!needle) {
+            throw SquirrelSystemBundleException("Missing `needle` parameter in `in_array` function");
+        }
+        if (!array) {
+            throw SquirrelSystemBundleException("Missing `array` parameter in `in_array` function");
```

Comment:
mb other message content, or add langs?
But should lang be added to this bundle? Or a language bundle?